src/clj/puppetlabs/services/protocols/jruby_pool.clj

```diff
@@ -0,0 +1,29 @@
+(ns puppetlabs.services.protocols.jruby-pool)
+
+(defprotocol JRubyPool
```

Comment:
These should have comments describing the inputs and expected behavior from a users perspective. (eg clarify https://github.com/puppetlabs/jruby-utils/pull/78/files#r353362968)